sonar-db/src/main/java/org/sonar/db/webhook/WebhookDeliveryMapper.java

```diff
@@ -27,6 +28,10 @@
   @CheckForNull
   WebhookDeliveryDto selectByUuid(@Param("uuid") String uuid);
 
+  List<WebhookDeliveryLiteDto> selectOrderedByComponentUuid(@Param("componentUuid") String componentUuid);
```

Comment:
name is ambiguous. I would understand from the name that result are ordered by componentUuid while they are actually ordered by created_at. It's not easy to find another name but at least some javadoc could be provided.